src/lib/style/components/_global-alert.scss

```diff
@@ -38,6 +38,7 @@
       grid-row: 1 / span 1;
       justify-self: end;
       padding-bottom: 6px;
+      color: #878E95;
```

Comment:
Close button should always be this grey.  https://sagebionetworks.jira.com/browse/SWC-6157